cpg_workflows/stages/seqr_loader.py

```diff
@@ -90,6 +90,11 @@ def queue_jobs(self, dataset: Dataset, inputs: StageInput) -> StageOutput | None
         """
         Annotate MT with genotype and consequence data for Seqr
         """
+        # only create dataset MTs for datasets specified in the config
+        eligible_datasets = config_retrieve(['workflow', 'write_mt_for_datasets'], default=[])
+        if dataset.name not in eligible_datasets:
```

Comment:
I can foresee a situation where someone (me) expects this to run and spends at least one whole day failing to realise this is a skip condition. Could you please add a logging statement in to say `Dataset X is being skipped as it doesn't appear in write_mt_for_datasets`